spec/feature/queue/pre_docket_spec.rb

```diff
@@ -124,6 +126,51 @@
 
           expect(vha_document_search_task.reload.status).to eq Constants.TASK_STATUSES.in_progress
         end
+
+        step "the 'Documents ready for Board Intake review' sends task to BVA Intake for review" do
+          User.authenticate!(user: vha_caregiver_user)
+
+          vha_document_search_task = VhaDocumentSearchTask.last
+          vha_document_search_task.update(assigned_to: vha_caregiver)
+          vha_document_search_task.update!(status: completed)
```

Comment:
```suggestion
          vha_document_search_task.update!(status: Constants.TASK_STATUSES.completed)
```
Believe you'll want to update this for your RSpec to pass